### PR TITLE
Hw 16

### PR DESCRIPTION
**Pull request by popovychyb**

### CODE REVIEW

File: src/main/java/mate/academy/internetshop/Main.java [hunk 1]
```diff
@@ -47,9 +46,9 @@ private static void initializeDb(ProductService productService,
...
         orderService.create(order2);
 
-        ShoppingCart shoppingCart1 = new ShoppingCart(productService.getAll(), admin);
-        ShoppingCart shoppingCart2 = new ShoppingCart(productService.getAll(), user);
-        shoppingCartService.create(shoppingCart1);
-        shoppingCartService.create(shoppingCart2);
+        //        ShoppingCart shoppingCart1 = new ShoppingCart(admin);
+        //        ShoppingCart shoppingCart2 = new ShoppingCart(user);
+        //        shoppingCartService.create(shoppingCart1);
+        //        shoppingCartService.create(shoppingCart2);
```
Comment (dmytro-hryhoruk) [inline src/main/java/mate/academy/internetshop/Main.java:52]: remove comments

File: src/main/java/mate/academy/internetshop/controllers/ProductsController.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.academy.internetshop.lib.Injector;
+import mate.academy.internetshop.model.Product;
+import mate.academy.internetshop.service.ProductService;
+
+public class ProductsController extends HttpServlet {
```
Comment (dmytro-hryhoruk) [inline src/main/java/mate/academy/internetshop/controllers/ProductsController.java:13]:
```suggestion
public class GetAllProductsController extends HttpServlet {
```

File: src/main/java/mate/academy/internetshop/controllers/RegistrationController.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        String name = req.getParameter("name");
+        String login = req.getParameter("login");
+        String password = req.getParameter("password");
+
+        userService.create(new User(name, login, password));
```
Comment (dmytro-hryhoruk) [inline src/main/java/mate/academy/internetshop/controllers/RegistrationController.java:30]: let's add confirming for password

File: src/main/java/mate/academy/internetshop/controllers/ShoppingCartController.java [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.academy.internetshop.lib.Injector;
+import mate.academy.internetshop.model.Product;
+import mate.academy.internetshop.service.ShoppingCartService;
+
+public class ShoppingCartController extends HttpServlet {
```
Comment (dmytro-hryhoruk) [inline src/main/java/mate/academy/internetshop/controllers/ShoppingCartController.java:13]:
```suggestion
public class GetShoppingCartController extends HttpServlet {
```

File: src/main/java/mate/academy/internetshop/controllers/ShoppingCartController.java [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+    public static final Long USER_ID = 1L;
+    private static final Injector INJECTOR = Injector.getInstance("mate.academy.internetshop");
+    private final ShoppingCartService shoppingCartService =
+            (ShoppingCartService) INJECTOR.getInstance(ShoppingCartService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        List<Product> products = shoppingCartService
+                .getAllProducts(shoppingCartService.getByUserId(USER_ID));
```
Comment (dmytro-hryhoruk) [inline src/main/java/mate/academy/internetshop/controllers/ShoppingCartController.java:23]: let's just use getter for products

File: src/main/java/mate/academy/internetshop/Main.java (hunk 1 above)
Comment (popovychyb) [reply src/main/java/mate/academy/internetshop/Main.java:52]: done

File: src/main/java/mate/academy/internetshop/controllers/ProductsController.java (hunk 2 above)
Comment (popovychyb) [reply src/main/java/mate/academy/internetshop/controllers/ProductsController.java:13]: done

File: src/main/java/mate/academy/internetshop/controllers/RegistrationController.java (hunk 3 above)
Comment (popovychyb) [reply src/main/java/mate/academy/internetshop/controllers/RegistrationController.java:30]: done

File: src/main/java/mate/academy/internetshop/controllers/ShoppingCartController.java (hunk 4 above)
Comment (popovychyb) [reply src/main/java/mate/academy/internetshop/controllers/ShoppingCartController.java:13]: done

File: src/main/java/mate/academy/internetshop/controllers/ShoppingCartController.java (hunk 5 above)
Comment (popovychyb) [reply src/main/java/mate/academy/internetshop/controllers/ShoppingCartController.java]: done

File: src/main/java/mate/academy/internetshop/controllers/UsersController.java [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.academy.internetshop.lib.Injector;
+import mate.academy.internetshop.model.User;
+import mate.academy.internetshop.service.UserService;
+
+public class UsersController extends HttpServlet {
```
Comment (dmytro-hryhoruk) [inline src/main/java/mate/academy/internetshop/controllers/UsersController.java:13]:
```suggestion
public class GetAllUsersController extends HttpServlet {
```

File: src/main/webapp/WEB-INF/views/cart/showCart.jsp [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+        <tr>
+<%--            <th>Id</th>--%>
+            <th>Name</th>
+            <th>Price</th>
+        </tr>
+        <c:forEach var="product" items="${products}">
+            <tr>
+<%--                <td>--%>
+<%--                    <c:out value="${product.id}"/>--%>
+<%--                </td>--%>
```
Comment (dmytro-hryhoruk) [inline src/main/webapp/WEB-INF/views/cart/showCart.jsp:19]: remove comments

**Pull request merged**